test/evergreen.yml

```diff
@@ -3572,7 +3572,7 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
-            export "PATH=/opt/mongodbtoolchain/v5/bin:$PATH"
+            export "PATH=/opt/mongodbtoolchain/v4/bin:$PATH"
```

Comment:
Hi Jie, this change reverts the memory model test to the V4 toolchain. Is that intentional?

Reply:
Thanks for spotting it, fixed it now.